OpenRA.Mods.Common/Activities/CaptureActor.cs

```diff
@@ -53,6 +57,7 @@ protected override void OnInside(Actor self)
 				if (actor.IsDead || capturable.BeingCaptured)
 					return;
 
+				var capturesInfo = captures.FirstOrDefault(c => !c.IsTraitDisabled).Info;
```

Comment:
Never use `.FirstOrDefault().<anything else>`!  You must always check that the result of `FirstOrDefault` is non-null before you try to call any properties or methods on it.

I suggest changing these few lines to:
```csharp
var activeCaptures = captures.FirstOrDefault(c => !c.IsTraitDisabled)

if (actor.IsDead || capturable.BeingCaptured || activeCaptures == null)
 	return;

var capturesInfo = activeCaptures.Info;
```